crates/cxx-qt-lib/build.rs

```diff
@@ -191,6 +194,10 @@ fn main() {
         "core/qvector/qvector_u64",
     ];
 
+    if qtbuild.version().major > 5 {
```

Comment:
wonder if it'd be useful to have this `version` exposed onto `CxxQtBuilder` so that you don't need to depend on `qt-build-utils` directly.

But then `CxxQtBuilder` doesn't actually create it's `QtBuild` until the `build()` method.

Something to think about separately though cc @LeonMatthesKDAB

Reply:
(note to future self: If we decide to do it later, might be worth adding TODOs so the implementer doesn't forget :-)